src/drivers/sdl/lv_sdl_window.c

```diff
@@ -208,6 +208,12 @@ void * lv_sdl_window_get_renderer(lv_display_t * disp)
     return dsc->renderer;
 }
 
+void * lv_sdl_window_get_window(lv_display_t * disp)
```

Comment:
This is a public function, so it would be good to assert the disp parameter before continue, otherwise there is a possibility of deferencing a null pointer.